hibernate-core/src/main/java/org/hibernate/type/TypeFactory.java

```diff
@@ -136,11 +136,11 @@ public Type type(Class<Type> typeClass, Properties parameters) {
 	public static void injectParameters(Object type, Properties parameters) {
 		if ( ParameterizedType.class.isInstance( type ) ) {
 			if ( parameters == null ) {
-				ConcurrentHashMapBackedProperties	concurrentParams = new ConcurrentHashMapBackedProperties(parameters);
-				( (ParameterizedType) type ).setParameterValues( concurrentParams );
+				( (ParameterizedType) type ).setParameterValues( EMPTY_PROPERTIES );
```

Comment:
That's why we need a replicating test case. Otherwise, how do you know you are actually fixing an issue? This last commit is the perfect example of why we need a replicating test case.

Reply:
@vladmihalcea The JIRA link and the replicating test case is already included in the pull request description. The last commit was made to rectify a miscommit i had made in the original.

Reply:
Thanks. I'll investigate it next week and let you know of my findings.